### ISSUE TITLE
false positive on 007105, /.well-known/core: core file...

### ISSUE THREAD

**Issue by digininja** [sullo/nikto#505]
If I'm reading the check correctly, it matched on the rt= in this line:

```
var Tawk_API=Tawk_API||{}, Tawk_LoadStart=new Date();
```

which is probably too generic but having said that, I've not seen this report before so maybe it isn't.

This is from the output file.

```
-----------------------------------------------------------------------
              Information
-----------------------------------------------------------------------
Test ID:    007105
OSVDB ID:   0   
Message:    /.well-known/core: core file may contain server info/links, per RFC-6690. See http://tools.ietf.org/html/rfc6690
Reason:     Content Match
-----------------------------------------------------------------------
              Request
-----------------------------------------------------------------------
GET /.well-known/core HTTP/1.1
Host: xxx
User-Agent: Mozilla/5.00 (Nikto/2.1.6) (Evasions:None) (Test:007105)
Connection: Keep-Alive

-----------------------------------------------------------------------
              Response
-----------------------------------------------------------------------
HTTP/1.1 404 Not Found
date: Fri, 01 Dec 2017 13:49:02 GMT 
server: Apache/2.4.18 (Ubuntu)
content-length: 20317
keep-alive: timeout=5, max=100
connection: Keep-Alive
content-type: text/html; charset=UTF-8
```

**Comment by sullo** [comment]
It's actually matching on two things:
   rt=  OR rf=

I'm a little surprised this doesn't FP more than it does; likely it doesn't because many web servers will error on the .well-known before processing any further.

The coap spec this file is based on is a little unclear on whether the strings are requirements or just examples and I can't find any good examples.  

I'm going to add a > before each. It's not foolproof but it should cut down the likelihood just a bit more?

**Comment by sullo** [comment]
Oye, put #70 in by mistake but this was fixed with commit 988e14c..a7c8d1b

**Comment by digininja** [comment]
Great, thanks
